src/vmm/src/devices/virtio/iovec.rs

```diff
@@ -1018,7 +1031,12 @@ mod verification {
     #[kani::proof]
     #[kani::unwind(5)]
     #[kani::solver(cadical)]
-    #[kani::stub(IovDeque::push_back, stubs::push_back)]
+    // The `IovDeque` is defined as type alias in the kani module. Because of this
+    // we need to specify original type here for stub to work.
+    #[kani::stub(
+        crate::devices::virtio::iov_deque::IovDeque::push_back,
```

Comment:
nit: I think that importing `IovDeque` in the `kani` module "preamble", and not having to use all the path here, is slightly more readable.

Reply:
The reason for full path was that `IovDeque` was an alias for `IovDeque<256>`. Replaced these with another types.